component-test/settings.gradle

```diff
@@ -1,18 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Same here. This is not OK. services, api, and component are some sort of subprojects but are not actually subprojects. 
This could actually be a plus but it's preferable to open PRs for only one thing and focus on it. You could maybe do this in a follow up PR. Do you understand me??